test/unit/adapters/http.js

```diff
@@ -261,6 +261,34 @@ describe('supports http with nodejs', function () {
     });
   });
 
+  it('should support max body length', function (done) {
+    var data = Array(100000).join('ж');
+
+    server = http.createServer(function (req, res) {
+      res.setHeader('Content-Type', 'text/html; charset=UTF-8');
+      res.end();
+    }).listen(4444, function () {
+      var success = false, failure = false, error;
+
+      axios.post('http://localhost:4444/', {
+        data: data
+      }, {
+        maxBodyLength: 2000
+      }).then(function (res) {
+        success = true;
+      }).catch(function (err) {
+        error = err;
+        failure = true;
+      });
+
+      setTimeout(function () {
+        assert.equal(success, false, 'request should not succeed');
+        assert.equal(failure, true, 'request should fail');
```

Comment:
Can you assert for `error.code`? See https://github.com/follow-redirects/follow-redirects/blob/master/index.js#L27.

Reply:
I think the tests shouldn't rely upon third party libraries responses / errors. However we might want to assert for it if we want to.

Reply:
Yes, `follow-redirects` is our important dependency. Don't worry it will change unexpectedly. And it is only test script.

Reply:
I conform that `follow-redirects` error codes are constants. In fact, they were specifically created for axios.

Reply:
Okay cool, i just added assertions for error code and message.